content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,159 @@
+```
+
+{{% /tab %}} {{% tab header="Windows (`standalone.conf.bat`)" lang=Windows %}}
+
+```bat
+set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{< /tabpane >}}
+
+## Jetty
+
+To define the path to the Java agent, use the `-javaagent` argument:
+
+```shell
+java -javaagent:/path/to/opentelemetry-javaagent.jar -jar start.jar
+```
+
+If you use the `jetty.sh` file to start Jetty, add the following line to the
+\<jetty_home\>/bin/jetty.sh file:
+
+```shell
+JAVA_OPTIONS="${JAVA_OPTIONS} -javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+If you use the start.ini file to define JVM arguments, add the `javaagent`
+argument after the `--exec` option:
+
+```ini
+#===========================================================
+# Sample Jetty start.ini file
+#-----------------------------------------------------------
+--exec
+-javaagent:/path/to/opentelemetry-javaagent.jar
+```
+
+## Glassfish / Payara
+
+Add the path to the Java agent using the `asadmin` tool:
+
+{{< tabpane text=true >}} {{% tab Linux %}}
+
+```sh
+<server_install_dir>/bin/asadmin create-jvm-options "-javaagent\:/path/to/opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{% tab Windows %}}
+
+```powershell
+<server_install_dir>\bin\asadmin.bat create-jvm-options '-javaagent\:<Drive>\:\\path\\to\\opentelemetry-javaagent.jar'
+```
+
+{{% /tab %}} {{< /tabpane >}}
+
+You can also add the `-javaagent` argument from the Admin Console. For example:
+
+1.  Open the GlassFish Admin Console at <http://localhost:4848>.
+2.  Go to **Configurations > server-config > JVM Settings**.
+3.  Select **JVM Options > Add JVM Option**.
+4.  Enter the path to the agent:
+    `-javaagent:/path/to/opentelemetry-javaagent.jar`
+5.  **Save** and restart the server.
+
+Make sure that the domain.xml file in your domain directory contains a
+`<jmv-options>` entry for the agent.
+
+## Tomcat / TomEE
+
+Add the path to the Java agent to your startup script:
+
+{{< tabpane text=true persist=lang >}}
+
+{{% tab header="Linux (`<tomcat_home>/bin/setenv.sh`)" lang=Linux %}}
+
+```sh
+CATALINA_OPTS="$CATALINA_OPTS -javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}}
+{{% tab header="Windows (`<tomcat_home>\bin\setenv.bat`)" lang=Windows %}}
+
+```bat
+set CATALINA_OPTS=%CATALINA_OPTS% -javaagent:"<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{< /tabpane >}}
+
+## WebLogic
+
+Add the path to the Java agent to your domain startup script:
+
+{{< tabpane text=true persist=lang >}}
+
+{{% tab header="Linux (`<domain_home>/bin/startWebLogic.sh`)" lang=Linux %}}
+
+```sh
+export JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}}
+{{% tab header="Windows (`<domain_home>\bin\startWebLogic.cmd`)" lang=Windows %}}
+
+```bat
+set JAVA_OPTIONS=%JAVA_OPTIONS% -javaagent:"<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
+
+{{% /tab %}} {{< /tabpane >}}
```

Comment:
Unfortunately, this renders poorly in the live site. Is there a way to shorten the text in the tabs?

<img width="766" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/6309070/fbe4cf0d-0e5b-46a9-bbc1-a8b028c13ef3">

Reply:
Yeah, when the tab name is too wide, this isn't optimal on mobile. I think that we do in other pages is add the path in the tab content.

Here's an example of how to fix this: https://github.com/open-telemetry/opentelemetry.io/pull/3459/files#r1383733272